### PR TITLE
format error strings to start with small caps

### PR DESCRIPTION
**Pull request by Madhu-1**
as per golang [documentation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings) golang [wiki](https://github.com/golang/go/wiki/Errors) error string should not be capitalized.

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

File: glusterd2/volume/volume-utils.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ func GetBrickMountRoot(brickPath string) (string, error) {
 	if mntSt := mntStat.Sys().(*syscall.Stat_t); brickSt.Dev == mntSt.Dev {
 		return "/", nil
 	}
-	return "", errors.New("Failed To Get Mount Root")
+	return "", errors.New("failed To Get Mount Root")
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply glusterd2/volume/volume-utils.go]: done

File: glusterd2/volume/volume-utils.go [hunk 2]
```diff
@@ -153,7 +153,7 @@ func GetBrickMountInfo(mountRoot string) (*Mntent, error) {
 			return entry, nil
 		}
 	}
-	return nil, errors.New("Mount Point Not Found")
+	return nil, errors.New("mount Point Not Found")
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply glusterd2/volume/volume-utils.go]: done

File: plugins/smartvol/bricksplanner/planner.go [hunk 3]
```diff
@@ -108,7 +108,7 @@ func getBricksLayout(req *smartvolapi.Volume) ([]smartvolapi.Subvol, error) {
 
 	subvolplanner, exists := subvolPlanners[req.SubvolType]
 	if !exists {
-		return nil, errors.New("Sub volume type not supported")
+		return nil, errors.New("sub volume type not supported")
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply plugins/smartvol/bricksplanner/planner.go]: done

File: pkg/sunrpc/errors.go [hunk 4]
```diff
@@ -38,17 +38,17 @@ func (e ErrProgMismatch) Error() string {
 // Given that the remote server accepted the RPC call, following errors
 // represent error status of an attempt to call remote procedure
 var (
-	ErrProgUnavail = errors.New("Remote server has not exported program")
-	ErrProcUnavail = errors.New("Remote server has no such procedure")
-	ErrGarbageArgs = errors.New("Remote procedure cannot decode params")
-	ErrSystemErr   = errors.New("System error on remote server")
+	ErrProgUnavail = errors.New("remote server has not exported program")
+	ErrProcUnavail = errors.New("rmote server has no such procedure")
```
[the remark this answers is not in the document]
Comment (Madhu-1) [reply pkg/sunrpc/errors.go]: done

**Comment by prashanthpai** [comment]
retest this please

File: glustercli/cmd/georep.go [hunk 5]
```diff
@@ -158,7 +158,7 @@ func parseRemoteData(data string) (string, string, string, error) {
 	remotehostvol := strings.Split(data, "::")
```
Comment (prashanthpai) [inline glustercli/cmd/georep.go]: The commit message title should be limited to 50 characters or so. Unfortunately, Github doesn't allow me to review commit messages. Please note this in future commits and PRs you send.

**Comment by Madhu-1** [comment]
Thanks, @prashanthpai missed running functional test cases. Fixed issue.

**Comment by prashanthpai** [comment]
retest this please

File: plugins/smartvol/rest.go [hunk 6]
```diff
@@ -28,11 +28,11 @@ func validateVolCreateReq(req *smartvolapi.VolCreateReq) error {
 	}
 
 	if req.Transport != "" && req.Transport != "tcp" && req.Transport != "rdma" {
-		return errors.New("Invalid transport. Supported values: tcp or rdma")
+		return errors.New("invalid transport. Supported values: tcp or rdma")
 	}
 
 	if req.Size < minVolumeSize {
-		return errors.New("Invalid Volume Size, Minimum size required is " + strconv.Itoa(minVolumeSize))
+		return errors.New("invalid Volume Size, Minimum size required is " + strconv.Itoa(minVolumeSize))
```
Comment (rishubhjain) [inline plugins/smartvol/rest.go]: Volume Size -> volume size and a full-stop instead of comma. Can be addressed in another PR

**Pull request merged**